scripts/release/rpm/test_azurelinux_in_docker.sh

```diff
@@ -14,7 +14,7 @@ time az self-test
 time az --version
 
 cd /azure-cli/
-python -m pip install --upgrade pip setuptools
+python -m pip install setuptools
```

Comment:
Why `--upgrade` is removed? This will cause `setuptools` not being updated.

Reply:
`setuptools` is not installed, so `--upgrade` is not required.